subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/TaskDefinitionIntegrationTest.groovy

```diff
@@ -105,25 +102,24 @@ class TaskDefinitionIntegrationTest extends AbstractIntegrationSpec {
 
     def "can define tasks using task method expression"() {
         buildFile << """
-            ext.a = 'a' == 'b' ? null: task(withAction) << { }
+            ext.a = 'a' == 'b' ? null: task(withAction) { }
             a = task(nothing)
             a = task(emptyOptions())
             ext.taskName = 'dynamic'
-            a = task("\$taskName") << { }
+            a = task("\$taskName") { }
             a = task('string')
-            a = task('stringWithAction') << { }
+            a = task('stringWithAction') { }
             a = task('stringWithOptions', description: 'description')
-            a = task('stringWithOptionsAndAction', description: 'description') << { }
+            a = task('stringWithOptionsAndAction', description: 'description') { }
             a = task(withOptions, description: 'description')
-            a = task(withOptionsAndAction, description: 'description') << { }
```

Comment:
❌ Missing `doLast`.

Please add the `doLast` statements. If you don't, then all those tasks won't have an action. And I don't think that is intended.

Reply:
I added the missing `doLast {}` actions in e2fbc490603.